Cargo.toml

```diff
@@ -9,5 +9,5 @@ name = "_odata_query"
 crate-type = ["cdylib"]
 
 [dependencies]
-chumsky = "0.9.2"
```

Comment:
I started with the `chumsky` framework (c7c2bd6c42ee8f318eb2dabe80a5fcc4d0a27e57), but found `nom` a bit easier, clearer and more stable.